alcs-frontend/src/app/services/notice-of-intent/decision-v2/notice-of-intent-decision-v2.service.spec.ts

```diff
@@ -214,4 +243,31 @@ describe('NoticeOfIntentDecisionV2Service', () => {
 
     expect(httpClient.get).toHaveBeenCalledTimes(1);
   });
+
+  it('should fetch and load a decision', async () => {
+    httpClient.get.mockReturnValue(
+      of([
+        {
+          fileNumber: '1',
+        },
+      ]),
+    );
+
+    await service.loadDecision('uuid');
+
+    expect(httpClient.get).toHaveBeenCalledTimes(1);
+  });
+
+  it('should show a toast message if load decision fails', async () => {
+    httpClient.get.mockReturnValue(
+      throwError(() => {
+        new Error('');
+      }),
+    );
+
+    const res = await service.loadDecision('uuid');
+
+    expect(httpClient.get).toHaveBeenCalledTimes(1);
```

Comment:
Do we need to check if `get` was called? What if this function fails before `get` is called.